client/src/components/StudentHelp.jsx

```diff
@@ -1,11 +1,64 @@
 import "./GeneralStyle.css"
 import circles from "../assets/circles.png";
 import dog from "../assets/questionDog.png";
-import { Box } from '@chakra-ui/react'
-
-export const StudentHelp = ({ currQuestion }) => {
+import { useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import { Box, useToast } from '@chakra-ui/react'
 
+export const StudentHelp = ({ currQuestion, selectedCourse, isTa, netId }) => {
+  
+  let navigate = useNavigate();
+  const routeChange = () => {
+    let path = `/login`;
+    navigate(path);
+  }
+  const toast = useToast();
   // TODO: make API call before loading
+  useEffect(() => {
+    // Define a function that makes the API call and updates the data state
+    const fetchData = async () => {
+      const response = await fetch(`http://localhost:3001/student/${selectedCourse}/${netId}`, {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+        }
+      });
+      try {
+        if(response.ok){
+          const responseData = await response.json();
+          console.log(responseData);
+          if(responseData.helped){
+            toast({
+              title: 'Thanks for Q\'dubbin',
+              description: "Login again to queue up!",
+              status: 'info',
+              duration: 10000,
+              isClosable: true,
+            });
+            routeChange();
+          }
+        }
+      } catch (error) {
+        console.error(error);
+      }
+
+
+      // INTERVAL GET HELPED STATUS
```

Comment:
are these TODOs?

Reply:
they are yes! I probably should have labeled them as so, but since I know I will be working on this next, I sort of just assumed.